packages/mdc-top-app-bar/component.ts

```diff
@@ -57,19 +61,49 @@ export class MDCTopAppBar extends MDCComponent<MDCTopAppBarBaseFoundation> {
     this.scrollTarget_ = window;
   }
 
+  initialSyncWithDOM() {
+    this.handleNavigationIconClick_ = this.foundation_.handleNavigationClick.bind(this.foundation_);
```

Comment:
`this.handleNavigationIconClick_`* to be same as foundation's callbak method name.